snuba/web/converters.py

```diff
@@ -1,15 +1,12 @@
 from werkzeug.routing import BaseConverter
 
 from snuba.datasets.dataset import Dataset
-from snuba.datasets.factory import get_dataset
+from snuba.datasets.factory import get_dataset, get_dataset_name
 
 
 class DatasetConverter(BaseConverter):
     def to_python(self, value: str):
         return get_dataset(value)
 
     def to_url(self, value: Dataset):
-        # This isn't needed right now, but should be straightforward to
-        # implement if necessary once there is a way to map a dataset instance
-        # back to it's name. (Basically, the inverse of ``get_dataset``.)
-        raise NotImplementedError
+        return get_dataset_name(value)
```

Comment:
I was about to complain about missing return type annotations and then realized I also missed them completely. 😞 

Would you mind adding them? 🙃